src/app/pages/home/home.jsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var StoreMixin = require('fluxible').StoreMixin;
+var GoogleMapsStore = require('../../maps/google-maps-store');
+var NavLink = require('../../nav/nav-link');
+var loadGoogleMaps = require('../../maps/load-google-maps');
+
+/**
+ * Home page.
+ */
+
+module.exports = React.createClass({
+  mixins: [StoreMixin],
+  statics: {
+    storeListeners: [GoogleMapsStore]
+  },
+  getInitialState: function () {
+    var venues = this.props.context.getStore('venue').getVenues();
+
+    return {
+      venues: venues
+    }
+  },
+  componentDidMount: function () {
+    var mapsIsReady = this.props.context.getStore('google-maps').isReady();
+    if (mapsIsReady) {
+      this.renderMap();
+    } else {
+      loadGoogleMaps(this.props.context);
```

Comment:
Does `componentDidMount` get called multiple times? I'm not sure about the flow here. React seems to imply that [it only gets called once](http://facebook.github.io/react/docs/component-specs.html#mounting-componentdidmount), so how could this both `loadGoogleMaps` and `renderMap`?